docs/0.2/creating_schemas.md

```diff
@@ -1,4 +1,4 @@
-# Creating Schemas
+# Creating and Updating Schemas
```

Comment:
We use the imperative everywhere else in the docs so I don't think we should change that here